justfile

```diff
@@ -405,32 +405,15 @@ db-migrate +ARGS:
     export $(cat .env.db.dev | xargs)
     cargo sqlx migrate --source "core/src/storage/migrations" {{ARGS}}
 
-web-ingestion-up: build-ingestion-service
-    #!/usr/bin/env bash
-    set -eux -o pipefail
+web-dev-up:
+    #!/usr/bin/env -S bash -eux -o pipefail
+    rm "$RUST_WORKSPACE/web-api/assets" || :
```

Comment:
❓ doesn't this need to be `-r` to delete it completely, or is a simple `rm` sufficient if this is a link?

Reply:
yes, `rm` is sufficient and I want it to fail if it's not a link